docs/conf.py

```diff
@@ -22,7 +22,7 @@
 
 # -- Project information -----------------------------------------------------
 project = 'QuakeMigrate'
-copyright = '2020, QuakeMigrate developers'
+copyright = '2020-2021, QuakeMigrate developers'
```

Comment:
```suggestion
 copyright = '2020–2021, QuakeMigrate developers'
```
en dash - really a question of our pedanticism...

Reply:
We'll do this in 2022 - not worth the faff.

Reply:
We'll do this in 2022. Not worth the faff